src/turbomind/models/llama/LlamaV2.h

```diff
@@ -48,6 +48,7 @@ class LlamaV2 {
         RequestQueue                          request_queue;
         std::shared_ptr<Barrier>              barrier;
         bool                                  abort;
+        std::atomic<size_t>                   free_size{size_t(-1)};
```

Comment:
Although [SIZE_MAX](https://en.cppreference.com/w/c/types/size_t) and `size_t(-1)` both represent `18446744073709551615` here, `SIZE_MAX` will be more intuitive. If possible, it is recommended to change it to `SIZE_MAX`.